security/README.md

```diff
@@ -0,0 +1,22 @@
+Security
+========
+
+Security is important, and you can't practice these guidelines without
+understanding them. Make sure you understand each guideline, why it exists, and
+how to follow it.
+
+Failing to follow these guidelines will likely put you, your team, and your
+deployed services at risk of compromise or loss of privacy.
+
+Using Passwords
+---------------
+* Use a unique password for every account you create.
+* Use a tool such as [1password](https://agilebits.com/onepassword) to generate random passwords.
+* Avoid sharing passwords in plaintext. If you must share a password in Slack,
+  delete the message immediately after you confirm it has been received.
```

Comment:
I'm not totally sold on Keybase after talking with George a bit about it, but we could definitely level up our PGP usage in general. For now I think I'd actually prefer sharing via 1password if necessary